x-pack/platform/plugins/shared/ml/common/types/storage.ts

```diff
@@ -16,6 +20,7 @@ export const ML_ANOMALY_EXPLORER_PANELS = 'ml.anomalyExplorerPanels';
 export const ML_NOTIFICATIONS_LAST_CHECKED_AT = 'ml.notificationsLastCheckedAt';
 export const ML_OVERVIEW_PANELS = 'ml.overviewPanels';
 export const ML_ELSER_CALLOUT_DISMISSED = 'ml.elserUpdateCalloutDismissed';
+export const ML_ACTIVE_MODEL_DEPLOYMENTS = 'ml.trainedModels.activeDeployments';
```

Comment:
I reckon this name is misleading. You use this prop for scheduling deployments, not storing existing deployments in the cluster. 
```suggestion
export const ML_SCHEDULED_MODEL_DEPLOYMENTS = 'ml.trainedModels.scheduledModelDeployments';
```

Reply:
Done in https://github.com/elastic/kibana/pull/205699/commits/e88fd196e0ef0d97160702a507515c570429bb05